.travis.yml

```diff
@@ -38,6 +38,8 @@ install:
   - pip install unidecode
   - pip install pycountry
   - pip install suds-jurko==0.6
+  - pip install genshi>=0.7
```

Comment:
Could you add this in manifest external dependencies ?

Reply:
it's a dependency of the seur package, the module don't uses it and if you install the seur package with pip or another toolm genshi it will be installed.